src/components/EditEvent/EventForm.tsx

```
@@ -1,83 +1,260 @@
+  const [formFields, setFormFields] = useState<FormFieldProps[]>(getFormFields(formValue));
   const [requiredByIndex, setRequiredByIndex] = useState<Record<number, boolean>>(() =>
     Object.fromEntries(formFields.map((_, i) => [i, true])),
   );
 
+  const createEventMutation = useCreateEventMutation();
+  const updateEventMutation = useUpdateEventMutation();
+
+  
+  // 신청 기간이 지났는지 확인하는 함수
+  const isApplicationPeriodExpired = () => {
+    if (!formValue?.applicationPeriod?.startDate || !formValue?.applicationPeriod?.endDate) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)